paasta_tools/check_chronos_jobs.py

```diff
@@ -200,13 +199,17 @@ def main():
             except utils.NoDeploymentsAvailable:
                 print(utils.PaastaColors.cyan("Skipping %s because no deployments are available" % service))
                 continue
-            sensu_output, sensu_status = sensu_message_status_for_jobs(
-                chronos_job_config=chronos_job_config,
-                service=service,
-                instance=instance,
-                cluster=cluster,
-                job_state_pairs=job_state_pairs
-            )
+            try:
+                sensu_output, sensu_status = sensu_message_status_for_jobs(
+                    chronos_job_config=chronos_job_config,
+                    service=service,
+                    instance=instance,
+                    cluster=cluster,
+                    job_state_pairs=job_state_pairs
+                )
+            except ValueError:
```

Comment:
This hints at the need for some refactoring. We're having to catch an exception that isn't even thrown in this function, but a function *it* depends on. I don't really like relying on that implementation detail for this logic.

maybe instead of this you could do:

```
fresh = job_state_pairs[0][1] == chronos_tools.LastRunState.NotRun if job_state_pairs else False
if not fresh:
   sensu_output, sensu_status = sensu_message_status_for_jobs(
                     chronos_job_config=chronos_job_config,
                     service=service,
                     instance=instance,
                     cluster=cluster,
                     job_state_pairs=job_state_pairs
                 )
      ...
```

I'm not completely happy with this here, but atleast now you're not tightly coupled to the implementation of ```sensu_message_status_for_jobs```